hooks/useProposalVotes.tsx

```diff
@@ -100,12 +103,12 @@ export default function useProposalVotes(proposal?: Proposal) {
     voteThresholdPct,
     yesVotePct,
     yesVoteProgress,
-    yesVoteCount,
-    noVoteCount,
+    yesVoteCount : Math.floor(yesVoteCount * scalingFactor),
```

Comment:
if you are going to add custom logic to generic calculations like this, it should at least be VERY explicit at each stage that A) scalingFactor is a Pyth thing and B) it's going to == 1 if Pyth isn't being used. Even more explicit would be to make it simply undfined if pyth isn't used, and then in these lines write `(pythScalingFactor || 1)`.

Reply:
Solved